src/enums/routeNames.ts

```diff
@@ -41,4 +41,11 @@ export enum RouteNames {
   AMALG_REG_PEOPLE_ROLES = 'amalg-reg-people-roles',
   AMALG_REG_REVIEW_CONFIRM = 'amalg-reg-review-confirm',
   AMALG_REG_SHARE_STRUCTURE = 'amalg-reg-share-structure',
+
+  // Continuation In route names
+  CONTINUATION_IN_BUSINESS_HOME = 'continuation-in-business-home',
+  CONTINUATION_IN_BUSINESS_BC = 'continuation-in-business-bc',
+  CONTINUATION_IN_PEOPLE_ROLES = 'continuation-in-people-roles',
+  CONTINUATION_IN_REVIEW_CONFIRM = 'continuation-in-review-confirm',
+  CONTINUATION_IN_SHARE_STRUCTURE = 'continuation-in-share-structure',
```

Comment:
Looks fine to me. Re:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/9a165258-fd2d-402a-8bb0-764cb29d2c47)